### PR TITLE
Updated SDI option to select SDI 1 or SDI 2

### PR DESCRIPTION
**Pull request by chrisrouse**
There are now two options to select either SDI 1 or SDI 2.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ function instance(system, id, config) {
 	self.command = null;
 
 	self.video_sources = [
-		{ id: 'SDI', label: 'SDI' },
+		{ id: 'SDI1', label: 'SDI 1' },
```
Comment (haakonnessjoen) [inline index.js:17]: You are missing the original "SDI" source

Comment (chrisrouse) [reply index.js:17]: Finally back at my computer long enough to take a look at this. Does it still need "SDI" in addition to "SDI 1" and "SDI 2" to work properly with the standard Teranex?

**Comment by gabonacorp** [comment]
Hi Chris,

Do you have any update in this issue? I have a teranex AV as well with two SDI inputs, I downloaded Companion 1.4 and 2.0 beta and the sdi input select still doesn't have two sdi options. 

Thank you in advance for your any effort on this!

best,

Andras

**Comment by gabonacorp** [comment]
Hi folks,

I just want to bump this issue. Teranex AV has two sdi input so the module needs to be specify which one should be selected. 

I made tests with 1.4 and 2.0 but input selection for SDI not working. HDMI, composit, optical, everything is perfect but for an sdi input command the devince is not responds.

Could you solve this somehow? Our company has a bunch of Teranex AV and it would be really helpful if I can select the inputs with my streamdeck.

Thanks for all the effort!!

Best,

Andras

File: index.js (hunk 1 above)
Comment (gabonacorp) [reply index.js:17]: yes, it needs. without that nothing happens.

**Review by gabonacorp — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (gabonacorp) [reply index.js:17]: it needs something like this:

self.video_sources = [
		{ id: 'SDI', label: ‘SDI (Express)' },
		{ id: 'SDI1', label: ‘SDI1 (AV)' },
		{ id: 'SDI2', label: ‘SDI2 (AV)' },
		{ id: 'HDMI', label: 'HDMI' },
		{ id: 'Composite', label: 'Composite' },
		{ id: 'Component', label: 'Component' },
		{ id: 'Optical', label: 'Optical' }
	];

Comment (haakonnessjoen) [reply index.js:17]: Chris, do you want to fix this so I can pull this? There are several people who want to see this fix :)

Comment (gabonacorp) [reply index.js:17]: bump this thread. I hardly need this, please guys...

**Comment by chrisrouse** [comment]
I'm working on getting my development environment setup again, but if anyone else wants to work on this or submit changes, it's not going to hurt my feelings. We got the part working that we needed for our event last year and don't have any other hardware to use for testing to confirm if it works with other versions of Teranex.

File: index.js (hunk 1 above)
Comment (josephdadams) [reply index.js:17]: @gabonacorp how can I help? Is it really just a matter of changing the `video_sources` array as you've requested?

**Comment by gabonacorp** [comment]
I tested it via telnet with booth the hardware. With SDI, SDI1 and SDI2 options all the versions are working properly. Teranex Express needs SDI, Teranex AV needs SDI1 and SDI2.

File: index.js (hunk 1 above)
Comment (gabonacorp) [reply index.js:17]: @josephdadams yes. if you can use my lines what I wrote on April 1 every Teranex devices gonna works properly. Teranex AV has two SDI input so it needs to be defined if it's SDI1 or SDI2. 

Every other commands are the same for booth versions of teranex.

**Pull request closed without merging**